userspace/libsinsp/sinsp.cpp

```diff
@@ -2708,6 +2660,27 @@ void sinsp::set_proc_scan_log_interval_ms(uint64_t val)
 	m_proc_scan_log_interval_ms = val;
 }
 
+void sinsp::set_sinsp_stats_v2_enabled()
+{
+	m_sinsp_stats_v2 = std::make_unique<sinsp_stats_v2>();
```

Comment:
@FedeDP forgot to add checks if it's NULL in case you would call it multiple times. Will re-push, but I'll wait for your feedback first in case there is more.